### ISSUE TITLE
Start New Report button was not working in Chrome when I start from https://www.w3.org/WAI/test-evaluate/conformance/

### ISSUE THREAD

**Issue by lakeen** [w3c/wai-wcag-em-report-tool#93]
When I start from https://www.w3.org/WAI/test-evaluate/conformance/ then select "WCAG-EM Report Tool: Website Accessibility Evaluation Report Generator" link there appears to be an additional #/ [https://www.w3.org/WAI/eval/report-tool/#/]

Then, if I select Start New Report, nothing happens except the url changes: https://www.w3.org/WAI/eval/report-tool/evaluation/define-scope 

I have to hit refresh to get the display to change and get to Step 1.

It appears to me that the issue is on https://www.w3.org/WAI/test-evaluate/conformance/ and not https://www.w3.org/WAI/eval/report-tool/

**Comment by shawna-slh** [comment]
Laura, Thank you very much for your troubleshooting and reporting.

Looks like the previous version of the tool used the # and the redesign that we launched a couple months ago does not -- but we didn't get that updated throughout the site.

I should be able to do that this afternoon.

**Comment by shawna-slh** [comment]
Hi @SteveALee @hidde 

The link we used for the previous version of the WCAG-EM Report Tool was
> https://www.w3.org/WAI/eval/report-tool/#/

That gets you to a version of the tool, but it is buggy.

The link since we launched the redesign is without the #:
> https://www.w3.org/WAI/eval/report-tool/

I corrected the links on the WAI website. However, I imagine there are others links in the world - people's bookmarks, etc.

I was going to set a redirect, and thought I’d check with you first if any problems with setting that redirect?

**Comment by SteveALee** [comment]
That doesn't seem to be related to the original issue description. Am I getting mixed up?

I'm not an expert on URIs but it seems to me that the two should identify the same resource on our server and so no redirect is needed. I tried in all browser and curl and get the same page as far as I can tell. What differece do you see?

In explanation:

The `#` fragment is processed by the browser, not the server. That might point to any problems being a browser bug

To break it down I'd expect that `https://www.w3.org/WAI/eval/report-tool/#/` means:

- the server directory resource `https://www.w3.org/WAI/eval/report-tool/`
  - the GtHub pages static server resolves this to `/eval/report-tool/index.html` 
  - a mute point if ever there was one, but whatever, the right page is served up
- a fragment on the page of `/` 
  - a fragment can be any ASCII character 
  - the fragment doesn't exist as an element with id=`/` on our page - so is ignored by browser
